js/src/Components/Atoms/FormDialog.js

```diff
@@ -47,7 +47,7 @@ export const FormDialog = ({open, title, formContent, onClose, ...props}) => {
         color: '#fff',
         position: 'sticky',
         top: 0,
-        zIndex: 2
+        zIndex: 1000
```

Comment:
That is very high. Does it need to be that high?

Reply:
it needs to be 1 below the media preview container and i tried that several hundred and still didn't work. so i just went there :man_shrugging: